packages/koa-liveness-ping/README.md

```diff
@@ -0,0 +1,30 @@
+# `@shopify/koa-liveness-ping`
+
+A package for creating a liveness ping middleware for use with koa
+
+A liveness ping is a URL at which your application will respond with a `200` whenever your server is running. It can be used, for example, for liveness checks in kubernetes deployments.
+
+## Installation
+
+```bash
+$ yarn add @shopify/koa-liveness-ping
+```
+
+## Usage
+
+```typescript
+import Koa from 'koa';
+import livenessPing from '@shopify/koa-liveness-ping';
```

Comment:
`LivenessPing` => `ping`

Reply:
on second thought, I use `ping()` in the tests and it looks good :+1: